### ISSUE TITLE
'shared_with_me' parameter

### ISSUE THREAD

**Issue by orzechszek** [astrada/google-drive-ocamlfuse#625]
Hi,

I've noticed that 'shared_with_me' parameter was removed. Is there another way not to synchronize .shared folder?

**Comment by astrada** [comment]
No, because it would break the "Add shortcut to my Drive" feature. Now that Google removed "Add to my drive" for shared files, there is no other way to access them.

**Comment by SebastianGrans** [comment]
I don't use the "Add shortcut to Drive" feature so disabling `.shared` folder from being mounted wouldn't really brake anything for me. It would be nice if it was still available with a warning that it could break some files from working.
